mlir/lib/Transforms/ViewOpGraph.cpp

```diff
@@ -250,7 +256,13 @@ class PrintOpPass : public impl::ViewOpGraphBase<PrintOpPass> {
   /// Process a block. Emit a cluster and one node per block argument and
   /// operation inside the cluster.
   void processBlock(Block &block) {
-    emitClusterStmt([&]() {
+
+    // Prepare the name for the block node
+    std::string nodeName;
+    llvm::raw_string_ostream blockNameStr(nodeName);
+    block.printAsOperand(blockNameStr);
+    Node node;
+    node = emitClusterStmt([&]() {
```

Comment:
Why not on a single line?
```c++
Node node = emitClusterStmt( /* the rest ...*/
```

Reply:
No reason at all, actually, the variables is useless, removed altogether